experiment/run_experiment.py

```diff
@@ -402,15 +405,16 @@ def start(self):
             docker_image_url,
             '/bin/bash',
             '-c',
-            'rsync -r '
-            '"${EXPERIMENT_FILESTORE}/${EXPERIMENT}/input/" ${WORK} && '
-            'mkdir ${WORK}/src && '
-            'tar -xvzf ${WORK}/src.tar.gz -C ${WORK}/src && '
-            'PYTHONPATH=${WORK}/src python3 '
-            '${WORK}/src/experiment/dispatcher.py || '
-            '/bin/bash'  # Open shell if experiment fails.
+            'rsync -r ',
```

Comment:
I don't understand this change. This is supposed to be one argument and you made it many.